src/psyclone/transformations.py

```diff
@@ -111,22 +111,14 @@ def _validate(self, node_list):
 
         # Check that the proposed region contains only supported node types
         for child in node_list:
-            flat_list = [child] + child.walk(child.children, object)
+            flat_list = [item for item in child.walk([child], object)
+                         if type(item) != Schedule]
```

Comment:
pylint says you should use isinstance instead of type()

Reply:
done